delta/plugins/blazegraph/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/routes/BlazegraphViewsRoutes.scala

```diff
@@ -1,17 +1,17 @@
 package ch.epfl.bluebrain.nexus.delta.plugins.blazegraph.routes
 
+import akka.http.scaladsl.model.StatusCodes
 import akka.http.scaladsl.model.StatusCodes.Created
-import akka.http.scaladsl.model.{MediaType, MediaTypes, StatusCodes}
 import akka.http.scaladsl.server.Directives._
 import akka.http.scaladsl.server.Route
 import cats.implicits._
+import ResponseToSparql._
```

Comment:
This import shouldn't be needed

Reply:
It's needed because I can't make `ResponseToJsonType` extend `SparqlValueInstances`.

Reply:
Then defeats the purpose of the Magnet Pattern